apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -147,6 +148,10 @@ type VSHNPostgreSQLServiceSpec struct {
 	// PgBouncerSettings passes additional configuration to the pgBouncer instance.
 	PgBouncerSettings *sgv1.SGPoolingConfigSpecPgBouncerPgbouncerIni `json:"pgBouncerSettings,omitempty"`
 
+	// Disable connection pooling. This means that PgBouncer will be disabled and all connections will be made directly to the PostgreSQL instance.
+	// +kubebuilder:default=false
+	DisableConnectionPooling bool `json:"disableConnectionPooling,omitempty"`
```

Comment:
I think this setup is confusing. Having `PgBouncerSettings` and `DisableConnectionPooling` in the same object and with different names might confuse the end user. The best way in my opinion would be to put those fields in an object names `ConnectionPooling`. This will probably be a pain in the ass as it will break the current API. What we could also do, we could just name them the bool field `DisablePgBouncer`. @Kidswiss what's your take on that?

Reply:
I thought about that, but struct managing pgBouncer Settings is called: `*sgv1.SGPoolingConfigSpecPgBouncerPgbouncerIni` which imho should be reserved just for Ini file settings

Reply:
I don't understand what your comment has to do with the issue I am raising. Why do we have to care about `*sgv1.SGPoolingConfigSpecPgBouncerPgbouncerIni` internal struct ?

Reply:
there's also 
```

// SGPoolingConfigSpec defines model for SGPoolingConfigSpec.
type SGPoolingConfigSpec struct {
	// Connection pooling configuration based on PgBouncer.
	PgBouncer *SGPoolingConfigSpecPgBouncer `json:"pgBouncer,omitempty"`
}
```
which sounds perfectly, but then it would be possible to configure pgbouncer ini in few places, which is also weird unfortunate

therefore I decided to add additional field

Reply:
You can also just create your own struct `ConnectionPooling` with `DisableConnectionPooling` and `PgBouncerSettings ` as fields.

Reply:
but I can't just move `PgBouncerSettings` to another struct as such change would impact all our running instances, I really don't think we should introduce such api changes in simple boolean field. I can add it to the `PgBouncerSettings` - it has nothing to do with pgbouncer ini file, but let's say it's the closest to the subject. Also if someone wants to disable PgBouncer it's rather unlike to set any PgBouncer settings

Reply:
Have you carefully read my first comment?

> I can add it to the `PgBouncerSettings`

No, you cannot. You would have the same problem as you described.

Reply:
I'd go with just renaming the field to `DisablePgBouncer`. Introducing even more levels in our claims might also get very confusing at one point.